recipes/hyperts/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "HyperTS" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/DataCanvasIO/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c62c713408d10e119a69ac1b6bba9959790db26ba1a5e3c1411e1cd4d734cbf6
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - hypernets >=0.2.5.1, <0.2.6
+    - sktime >=0.8.1
+    - prophet ==1.0.1
+    - statsmodels >=0.12.1
+    - numpy >=1.16.5,<1.22
+    - pandas >=0.25.3
+    - scikit-learn >=0.22.1
+    - scipy <1.8.0
```

```suggestion
    - scipy <1.8.0
  run_constrained:
    - tensorflow >=2.0
```
This makes sure that if TensorFlow gets installed, it will be `tensorflow>=2.0`.